### PR TITLE
Adding example for ES6 Internationalization and Localization features

### PR DESCRIPTION
**Pull request by shaktilucifer**
#### Pre-Submission Checklist
- [x] Your pull request targets the `develop` branch.
- [ ] Branch starts with either `fix/` or `feature/`
- [x] All new and existing tests pass the command `npm test`.
- [ ] Any new files are included in the grunt tasks

#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [x] Tested changes locally.
- [ ] New tests added?
- [x] Closes currently open issue : Adds to #74

#### Full Description of your Changes
Added an example for ES6 Internationalization and Localization features

### CODE REVIEW

File: src/examples/internationalization-localization.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/* Language sensitive string comparison using Collator */
+// German 'ä' sorts with a
+// Swedish 'ä' sorts after z
+console.log(l10nDE.compare('ä','z'));// negative value
+console.log(l10nSV.compare('ä','z'));// positive value
+console.log(['ä','a','z'].sort(l10nDE.compare)); // ['a','ä','z']
+console.log(['ä','a','z'].sort(l10nSV.compare)); // ["a", "z", "ä"]
+
+/* Date Time Format */
+let l10nUS = new Intl.DateTimeFormat("en-US") // English - US
```
Comment (ageddesi) [inline src/examples/internationalization-localization.js]: missing semicolon

File: src/examples/internationalization-localization.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+let l10nGB = new Intl.DateTimeFormat("en-GB") // English - GB
```
Comment (ageddesi) [inline src/examples/internationalization-localization.js]: missing semicolon

File: src/examples/internationalization-localization.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+let l10nEG = new Intl.DateTimeFormat("ar-EG") // Arabic
```
Comment (ageddesi) [inline src/examples/internationalization-localization.js]: missing semicolon

File: src/examples/internationalization-localization.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+let date = new Date(Date.UTC(2016, 1, 19, 19, 19, 19));
+
+console.log(l10nUS.format(date)); // US follows month/day/year format
+console.log(l10nGB.format(date)); // GB follows day/month/year format
+console.log(l10nEG.format(date)); // Arabic uses thier own arabic digits
+
+/* Number Format */
+let l10nDEnum = new Intl.NumberFormat('de-DE');
+let l10nINnum = new Intl.NumberFormat('en-IN');
+let number = 9876532.10
```
Comment (ageddesi) [inline src/examples/internationalization-localization.js]: missing semicolon

**Comment by shaktilucifer** [comment]
Whoops! my bad. Don't know how i missed it.

**Comment by joshghent** [comment]
Thanks @shaktilucifer  for your PR! 🎉 👍 
Take a look at our other [issues](https://github.com/esfiddle/esfiddle/issues) if you would like or come chat with us on [gitter](https://gitter.im/esfiddle/Lobby)!

**Pull request merged**